index.js

```diff
@@ -1,43 +1,53 @@
-    'no-debugger':                 [`warn`],
-
-    'arrow-parens':                [`error`, `as-needed`],
-    'camelcase':                   [`error`, {'properties': `always`}],
-    'comma-dangle':                [`error`, `always-multiline`],
-    'comma-spacing':               [`error`, {'before': false, 'after': true}],
-    'eol-last':                    [`error`],
-    'eqeqeq':                      [`error`],
-    'indent':                      [`error`, 2, {SwitchCase: 1}],
-    'key-spacing':                 ['error', {beforeColon: false, afterColon: true, mode: `minimum`}],
-    'keyword-spacing':             [`error`],
-    'linebreak-style':             [`error`, `unix`],
-    'multiline-ternary':           [`error`, `never`],
-    'no-multi-spaces':             ['error', {exceptions: {Property: true }}],
-    'no-trailing-spaces':          [`error`],
-    'no-unused-expressions':       [`error`],
-    'no-use-before-define':        [`error`, {classes: false}],
-    'no-var':                      [`error`],
-    'object-curly-spacing':        [`error`, `never`],
-    'object-shorthand':            [`error`, `always`],
-    'prefer-const':                [`error`, {destructuring: `all`, ignoreReadBeforeAssign: true}],
-    'quotes':                      [`error`, `backtick`],
-    'semi':                        [`error`, `always`],
-    'sort-keys':                   [`off`],
-    'space-before-blocks':         [`error`, `always`],
+    'arrow-parens': [`error`],
+    'comma-dangle': [`error`, `always-multiline`],
+    'comma-spacing': [`error`, {before: false, after: true}],
+    'eol-last': [`error`],
+    eqeqeq: [`error`],
+    'key-spacing': [`error`, {beforeColon: false, afterColon: true, mode: `minimum`}],
+    'keyword-spacing': [`error`],
+    'linebreak-style': [`error`, `unix`],
+    'no-console': [WARN_IF_DEV, {allow: [`info`, `warn`, `error`, `assert`]}],
+    'no-control-regex': [`off`],
+    'no-debugger': [WARN_IF_DEV],
+    'no-multi-spaces': [`error`, {exceptions: {Property: true}}],
+    'no-prototype-builtins': [`off`],
+    'no-trailing-spaces': [`error`],
+    'no-unused-expressions': [`error`, {allowShortCircuit: true, allowTernary: true}],
+    'no-var': [`error`],
+    'object-curly-spacing': [`error`, `never`],
+    'object-shorthand': [`error`, `always`],
+    'prefer-const': [`error`, {destructuring: `all`, ignoreReadBeforeAssign: true}],
+    semi: [`error`, `always`],
+    'sort-keys': [`off`],
+    'space-before-blocks': [`error`, `always`],
     'space-before-function-paren': [`error`, {anonymous: `never`, named: `never`, asyncArrow: `always`}],
+    'no-use-before-define': [
+      `error`,
+      {
+        functions: false,
+        classes: false,
+      },
+    ],
+    'no-unused-vars': [`error`, {argsIgnorePattern: `^_`}],
+    camelcase: [`error`, {properties: `never`, ignoreDestructuring: true}],
+    quotes: [`error`, `backtick`],
```

Comment:
mostly sorted but not quite? at least make the whole list a-z if it's going to become a prettiered mess

Reply:
I can feel your love of prettier through this Ted. I didn't even try it was just happen-stance. I'll update.